### DISCUSSION TITLE
Access swagger text in react app

### DISCUSSION THREAD

**Discussion by Uday287** [scalar/scalar#601] [Q&A] [answered]
Hi all,
I am using the scalar/api-reference in my react app, just wanted to find out how I can access the contents of the swagger editor, if I wanted to export the specification text

**Comment by hanspagel** [comment]
Oh, yeah. We’ll expose an event or sth! I’ve created an issue #603

**Reply by Uday287** [reply]
Hi @hanspagel , thanks for taking this up.
Approximately how long could it take to come up with a solution?

**Reply by marclave** [reply, marked as answer]
Fixed in https://github.com/scalar/scalar/pull/605 ✨ version @scalar/api-reference@1.5.0

**Reply by Uday287** [reply]
Thanks a lot!!
